### PR TITLE
VTCCA-3971 title and heading match

### PR DESCRIPTION
**Pull request by michjanowicz**

### CODE REVIEW

File: app/views/propertyLinking/chooseEvidence.scala.html [hunk 1]
```diff
@@ -23,8 +23,12 @@
...
 @title = @{clientDetails.map(_.organisationId) match {
-    case Some(id)   => {messages("chooseEvidence.client.title")}
-    case _          => {messages("chooseEvidence.title")}
+    case Some(id) => {messages("chooseEvidence.client.title")}
+    case _ => {messages("chooseEvidence.title")}
+}}
+@label = @{clientDetails.map(_.organisationId) match {
+    case Some(id) => {messages("chooseEvidence.client.title")}
+    case _ => {messages("chooseEvidence.title")}
 }}
```
Comment (darrell-plant) [inline app/views/propertyLinking/chooseEvidence.scala.html:32]: `title` and `label` seem to be exactly the same - if so you don't need to duplicate this code

Comment (michjanowicz) [reply app/views/propertyLinking/chooseEvidence.scala.html:32]: Thanks, I will work on it

Comment (michjanowicz) [reply app/views/propertyLinking/chooseEvidence.scala.html]: Its done, thanks for feedback

File: app/views/propertyLinking/chooseEvidence.scala.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
                     @govukRadios(Radios(
                         fieldset = Some(Fieldset(
                             legend = Some(Legend(
-                                content = Text(messages(s"label.hasRatesBill")),
+                                content = Text(messages(title)),
```
Comment (darrell-plant) [inline app/views/propertyLinking/chooseEvidence.scala.html:44]: I think this should be ```
   content = Text(title)
```
?

Comment (michjanowicz) [reply app/views/propertyLinking/chooseEvidence.scala.html]: Done. Thanks

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by zahiriqbal** [comment]
👍

**Pull request merged**